packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -84,7 +84,7 @@ const DefaultPaymentLegalBlurb = () => (
         ),
         paypalPrivacyLink: (
           <a
-            href="https://paypal.com/privacy"
+            href="https://www.paypal.com/webapps/mpp/ua/privacy-full"
```

Comment:
I actually have no idea how this reverted back to https://paypal.com/privacy , as it was updated in #10074 ... (see [here](https://github.com/mozilla/fxa/pull/10074/files#diff-f8298171cc067c65c23e886c6d115f61da4000621aa7b060b8aa23dd425313dcL31)). @chenba any ideas?

My hypothesis is the localization ID. Can you change the localization ID here and in main.ftl to be safe?

Reply:
I was confused for a while, but that link is in two places.  This is not the same location as the one you linked in your comment.

Reply:
Oh right; same file but in two locations. Odd.